src/modules/apiLoadFirstSetup.js

```diff
@@ -1,15 +1,35 @@
-import api from '../utils/api'
+import { useDataQuery } from '@dhis2/app-runtime'
 import { NAMESPACE } from '../constants/data-store'
-import { getAuthority } from './getAuthority'
-
-export const apiFirstLoad = () => {
-    return getAuthority().then(hasAuthority => {
-        return api.getNamespaces().then(result => {
-            if (result.includes(NAMESPACE)) {
-                return { hasNamespace: true }
-            } else {
-                return { hasAuthority, hasNamespace: false }
-            }
-        })
-    })
+
+export const namespaceDataStoreQuery = {
+    dataStore: {
+        resource: 'dataStore',
+    },
+}
+
+export const authorityQuery = {
+    authority: {
+        resource: 'me/authorization/ALL',
```

Comment:
Should this be `authorization` or `authorities`?  Probably best to ask someone on the api team. I just wondered because I was checking out the new documentation and saw `authorities` documented. Seems they both work, but one may be preferred over the other.

Reply:
I've just asked, it seems both resources are identical. and `authorities` is preferable as it's more descriptive of what it is, but are identical. Should I change it to `authorities`?